webpack.dev.js

```diff
@@ -9,5 +10,79 @@ module.exports = merge(
         devServer: {
             host: '0.0.0.0',
         },
+        module: {
+            rules: [
+                // All global styles live in global.scss. Let these cascade.
+                {
+                    test: /global\.scss$/,
+                    exclude: /node_modules/,
+                    use: [
+                        'style-loader',
+                        'css-loader',
+                        {
+                            loader: 'sass-loader',
+                            options: {
+                                implementation: sass,
+                            },
+                        },
+                    ],
+                },
+                // Load the rest of the styles as (s)css modules.
+                {
+                    test: /\.scss$/,
+                    exclude: [/node_modules/, /global\.scss$/],
+                    use: [
+                        'style-loader',
+                        {
+                            loader: 'css-loader',
+                            options: {
+                                sourceMap: true,
+                                modules: true,
+                                importLoaders: 1,
+                                localIdentName: '[name]__[local]___[hash:base64:5]',
+                            },
+                        },
+                        {
+                            loader: 'postcss-loader',
+                            options: {
+                                sourceMap: true,
+                            },
+                        },
+                        {
+                            loader: 'sass-loader',
+                            options: {
+                                implementation: sass,
+                                sourceMap: true,
+                            },
+                        },
+                    ],
+                },
+                {
+                    test: /\.(gif|png|jpe?g|svg)$/i,
+                    use: [
+                        'file-loader',
```

Comment:
I've updated some packages to address some security vulnerabilities (already pushed to `dev`) and discovered a couple of things:
- `file-loader`, `url-loader` and `raw-loader` are deprecated in Webpack 5, in favor of Asset Modules (see https://webpack.js.org/guides/asset-modules/)
- it looks like `image-webpack-loader` is no longer maintained (it's the source of a bunch of security vulnerabilities and it hasn't been updated in 6 months)

Of note with Asset Modules is if you specify `asset` (rather than `asset/resource` or `asset/inline`), webpack will automatically use size to determine whether a specific resource should be inlined or spit out as a separate file. 

Regarding `image-webpack-loader`, I don't know why I used it (must have been in some boiler-plate getting-started tutorial for TS), but it's not actually required to *load* an image -- it just minimizes it. So I think we can just remove `image-webpack-loader`, ensure any images we use are already an appropriate size, and benefit from the removed security vulnerabilities :smile:

I've confirmed that the following does indeed work:
```
                {
                    test: /\.(gif|png|jpe?g|svg)$/i,
                    type: 'asset/resource',       <----- or 'asset' or 'asset/inline'
                },
```

Soooooo. If I'm understanding properly, we should be able to use Asset Modules for both images and the fonts you've added. Thoughts on all this?